grails-app/conf/BuildConfig.groovy

```diff
@@ -146,6 +144,20 @@ grails.project.dependency.resolution = {
 
         // for com.google.common
         compile 'com.google.guava:guava:12.0'
+
+        compile 'org.jxls:jxls:2.8.1' // TODO: This is the last version for java 7. After migration to Java 8 upgrade this to 2.9+
+        compile 'org.jxls:jxls-poi:1.0.9' // this one supports apache poi v3 (anything above is for v4, which is Java 8)
```

Comment:
left to its own devices, `jxls-poi` will add a dependency on a `jlxs` version that requires Java 8. Try `compile 'org.jxls:jxls-poi:1.0.9' { exclude "jxls" }` here?